package.json

```diff
@@ -131,7 +131,7 @@
     "webpack": "^5.65.0"
   },
   "peerDependencies": {
-    "@ember/test-helpers": "^2.0.0",
+    "@ember/test-helpers": "^3.0.0",
```

Comment:
```suggestion
    "@ember/test-helpers": "^3.0.3",
```
Only suggesting this change because there were some shenanigans with a few early versions of 3.0 and we're in the process of unpublishing/deprecating them ><

Reply:
resolved!